resources/assets/uad_lists.json

```
@@ -37452,5 +37461,212 @@
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.motorola.android.overlay.deviceconfig.manaus",
+    "list": "Oem",
+    "description": "Random configurations to basic things(powersaving, network, camera, etc.).",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Unsafe"
+  },
+  {
+    "id": "com.motorola.android.overlay.lhbm",
+    "list": "Oem",
+    "description": "Not sure if its for high brightness mode, still unknown\n'config_udfps_local_hbm_supported' 'true'",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Expert"
+  },
+  {
+    "id": "com.motorola.motolights",
+    "list": "Oem",
+    "description": "Moto Lights\nPeople recommend to remove it.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.motorola.android.settings.overlay.eqs",
+    "list": "Oem",
+    "description": "Overlay to refresh rate change in settings.\nNot sure if it's unused.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Expert"
+  },
+  {
+    "id": "com.motorola.sstservice",
+    "list": "Oem",
+    "description": "Spatial Sound\nProvides sst audio effects.\nI am not sure how to use this app.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.motorola.msimsettings.overlay",
+    "list": "Oem",
+    "description": "In code found: 'channel_force_single_sim'. Looks very unused.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.motorola.paramupdater",
+    "list": "Oem",
+    "description": "I am not sure what this app is for.\nCollects systeminfo, time, has notification about update.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.mediatek.presence",
+    "list": "Misc",
+    "description": "App used to IMS, RCS. IP Voice Call.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  {
+    "id": "com.mediatek.ims.rcsua.service",
+    "list": "Misc",
+    "description": "Needed to support IMS, RCS probably.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  {
+    "id": "com.telcel.contenedor",
+    "list": "Carrier",
+    "description": "Telcel app, Advertising company to get promotions.\nhttps://play.google.com/store/apps/details?id=com.telcel.contenedor",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.motorola.thermalservice",
+    "list": "Oem",
+    "description": "Motorola Thermal Service\nTurns off phone when it's too hot.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
```

```suggestion
    "removal": "Recommended"
```
This is an important aspect of the system. Why did you make it `Recommended`??

---

This app does not look very useful, not all moto have this app than just some. This app notifies you when the phone is too hot or turns off it, no more stuff.

---

Yes that's considered a vital part of the system actually. If this does not happen, your phone can explode or catch fire. This should be `Expert`.